Common/Tests/Utilities/Mocks/MockReplWindow.cs

```diff
@@ -320,9 +329,9 @@ public class MockReplWindow : IReplWindow {
         #endregion
     }
 
-#if DEV14_OR_LATER
     public static class ReplEvalExtensions {
-        public static Task<ExecutionResult> Initialize(this IReplEvaluator self, IReplWindow window) {
+#if DEV14_OR_LATER
+        public static Task<ExecutionResult> _Initialize(this IReplEvaluator self, IReplWindow window) {

```

Comment:
To avoid conflicting with the real Initialize method when the class implements it, since then we skip the extension method. Having a unique name for tests makes it more obvious that we're doing something strange here (aka. supporting VS 2013 and 2015 with the same code)